docs/reference/query-dsl/multi-match-query.asciidoc

```diff
@@ -247,273 +250,198 @@ number of `match` clauses.
 
-The `cross_field` type tries to solve these problems at query time by taking a
-_term-centric_ approach.  It first analyzes the query string into individual
-terms, then looks for each term in any of the fields, as though they were one
-big field.
+The `combined_fields` type can only work in term-centric mode on fields that
+have the same analyzer. Fields with the same analyzer are grouped together
+as in the example above, and scored as a single combined field. If there are
+multiple groups, they are combined with a `bool` query.
 
-A query like:
+For instance, if we have a `title` field which uses the standard analyzer,
+and `abstract` and `body` which both add the `stemmer` filter, then this query
 
 [source,console]
 --------------------------------------------------
 GET /_search
 {
   "query": {
     "multi_match" : {
-      "query":      "Will Smith",
-      "type":       "cross_fields",
-      "fields":     [ "first_name", "last_name" ],
-      "operator":   "and"
+      "query":      "information",
+      "type":       "combined_fields",
+      "fields":     [
+        "title", "abstract", "body"
+      ]
     }
   }
 }
 --------------------------------------------------
 
-is executed as:
+would be executed as
 
-    +(first_name:will  last_name:will)
-    +(first_name:smith last_name:smith)
+    (combined("information", fields:["title"]))
+    (combined("inform", fields:["abstract", "body"]))
 
-In other words, *all terms* must be present *in at least one field* for a
-document to match.  (Compare this to
-<<operator-min,the logic used for `best_fields` and `most_fields`>>.)
+In other words, `title` is treated as its own group, and `abstract` and `body`
+are grouped together and treated as a single field. These analysis groups are
+combined through a `should` boolean query, allowing either group to match.
+
+Having multiple groups is fine, but when combined with `operator` or
+`minimum_should_match`, it can suffer from the <<operator-min,same problem>>
+as `most_fields` or `best_fields`.
 
-That solves one of the two problems. The problem of differing term frequencies
-is solved by _blending_ the term frequencies for all fields in order to even
-out the differences.
+[[type-cross-fields]]
+==== `cross_fields`
 
-In practice, `first_name:smith` will be treated as though it has the same
-frequencies as `last_name:smith`, plus one. This will make matches on
-`first_name` and `last_name` have comparable scores, with a tiny advantage
-for `last_name` since it is the most likely field that contains `smith`.
+The `cross_fields` type also takes a term-centric approach to scoring, allowing
+each term to match any of the provided fields. It tries to pretend that the content
+had been indexed into one combined field by _blending_ the term statistics across
+fields. It also gives a slight scoring advantage to fields that frequently contain
+the term, under the assumption that these matches are most interesting.
```

Comment:
It would be nice to add a sentence explaining the difference between  `cross_fields` and` combined_fields` ,  and why `combined_fields` are superior.

Reply:
Thanks for taking a look! I added a couple sentences. When we deprecate `cross_fields` in a follow-up, they'll be pulled into a deprecation note.